source/index.html.md

```diff
@@ -67,175 +67,394 @@ Kittn expects for the API key to be included in all API requests to the server i
+        "d_param": "721253437254664138",
+        "is_stattrak": false,
+        "is_souvenir": false,
+        "rarity": 2,
+        "quality": 4,
+        "market_hash_name": "M4A1-S | Boreal Forest (Field-Tested)",
+        "stickers": [
+            {
+                "stickerId": 55,
+                "slot": 0,
+                "icon_url": "emskatowice2014/fnatic.7f37dae42f8afbd799b89f77334be023368ba27a.png",
+                "name": "Sticker | Fnatic | Katowice 2014",
+                "scm": {
+                    "price": 41000,
+                    "volume": 0
+                }
+            },
+            {
+                "stickerId": 55,
+                "slot": 1,
+                "wear": 0.09002881,
+                "icon_url": "emskatowice2014/fnatic.7f37dae42f8afbd799b89f77334be023368ba27a.png",
+                "name": "Sticker | Fnatic | Katowice 2014",
+                "scm": {
+                    "price": 41000,
+                    "volume": 0
+                }
+            },
+            {
+                "stickerId": 73,
+                "slot": 2,
+                "wear": 0.21217501,
+                "icon_url": "emskatowice2014/reason.d48f01f2758c2852ef32a68c49f7039ce211500a.png",
+                "name": "Sticker | Reason Gaming | Katowice 2014",
+                "scm": {
+                    "price": 118625,
+                    "volume": 0
+                }
+            },
+            {
+                "stickerId": 73,
+                "slot": 3,
+                "icon_url": "emskatowice2014/reason.d48f01f2758c2852ef32a68c49f7039ce211500a.png",
+                "name": "Sticker | Reason Gaming | Katowice 2014",
+                "scm": {
+                    "price": 118625,
+                    "volume": 0
+                }
+            }
+        ],
+        "tradable": 0,
+        "has_screenshot": true,
+        "scm": {
+            "price": 11,
+            "volume": 0
+        },
+        "item_name": "M4A1-S | Boreal Forest",
+        "wear_name": "Field-Tested",
+        "description": "It has been painted using a forest camouflage hydrographic.\\n\\n<i>The woods can be a dangerous place... never travel alone</i>",
+        "collection": "The Italy Collection",
+        "badges": []
+    },
+    "is_seller": false,
+    "min_offer_price": 7565,
+    "max_offer_discount": 1500,
+    "is_watchlisted": false,
+    "watchers": 0
 }
 ```
 
-This endpoint deletes a specific kitten.
+This endpoint retrieves a specific listing.
 
 ### HTTP Request
 
-`DELETE http://example.com/kittens/<ID>`
+`POST https://csgofloat.com/api/v1/listings`
 
-### URL Parameters
-
-Parameter | Description
---------- | -----------
-ID | The ID of the kitten to delete
+### Body Parameters
 
+Parameter | Default | Description | Optional
+--------- | --------| ----------- | ----------
+type | `buy_now` | Either `buy_now` or `auction` | NO
+asset_id | | The ID of the item to list | NO
+price | | `buy_now` price | NO
```

Comment:
Price is also annotated for auctions, which signifies the current bid price (if there is a bid), or the reserve price (if there isn't a bid)